calico_versioned_docs/version-3.26/releases.json

```diff
@@ -1,4 +1,56 @@
 [
+  {
+    "title": "v3.26.2",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "registry": "quay.io",
+      "version": "v1.30.3"
```

Comment:
@danudey Have you got a new operator version for us? 

And any other component version changes while you're at it? Thanks!

Reply:
Updated